docs/docs/telemetry.mdx

```diff
@@ -0,0 +1,122 @@
+---
+id: telemetry
+sidebar_label: Rasa Telemetry
+title: Rasa Telemetry
+---
+
+Rasa uses telemetry to report anonymous usage information. This information is
+essential help improve Rasa Open Source for all users.
+For the team working on Rasa Open Source it is important to understand how
+the product is used, it allows us to properly prioritize our research efforts
+and feature development.
+
+You will be notified about the telemetry reporting when running Rasa Open Source
+for the first time.
+
+## How to opt-out
+
+You can always opt-out from telemetry reporting using
```

Comment:
You can always opt-out => You can opt out of telemetry reporting at any time using...